doc/content/with_conda.md

```diff
@@ -39,13 +42,42 @@ If the above produces a `cardinal-opt` Cardinal executable, you can
 jump straight to [#running]. If you want to customize the
 build or were not successful with the above, please consult the detailed instructions
 that follow.
+
+!alert! tip title=Using OpenMC from your Conda Environment
+You can install OpenMC directly into your conda environment. See the "Using OpenMC from your Conda Environment" section for more details.
+!alert-end!
 !alert-end!
 
 ## Access
 
 !include access.md
 
-To get MOOSE's conda environment, follow the instructions [here](https://mooseframework.inl.gov/getting_started/installation/index.html).
+To get started, you'll need to create a dedicated conda environment for Cardinal. This ensures that all the necessary dependencies are isolated and managed correctly.
```

Comment:
I'm not a conda expert, but I don't think this is truly necessary? I'd prefer to avoid any customized instructions beyond just what MOOSE requires (but please educate me if there is an advantage to doing this).